### DISCUSSION TITLE
Is there a rule that can prohibit comments in use statements?

### DISCUSSION THREAD

**Discussion by Saeven** [squizlabs/PHP_CodeSniffer#3940] [Q&A]
e.g., stuff like this, which drives me nuts

```
use Doctrine\ORM\EntityManager;
//use http\Header;
use Laminas\View\Model\JsonModel;
```

**Comment by fredden** [comment]
There is a sniff which aims to detect and complain about PHP code which has been commented out: `Squiz.PHP.CommentedOutCode`
https://github.com/PHPCSStandards/PHP_CodeSniffer/blob/3.8.1/src/Standards/Squiz/Sniffs/PHP/CommentedOutCodeSniff.php
It's not specifically designed to detect commented-out `use` statements. I imagine writing a sniff to detect this case would be possible. Is your standard public and open to contributions?
